src/Admin/AbstractAdmin.php

```diff
@@ -2853,11 +2853,30 @@ public function getDashboardActions()
             ];
         }
 
-        $actions = $this->configureDashboardButtons($actions);
+        $actions = $this->configureDashboardActions($actions);
+
+        // NEXT_MAJOR: Remove this code.
+        $newActions = $this->configureDashboardButtons($actions);
+        if ($newActions !== $actions) {
+            @trigger_error(
+                'configureDashboardButtons() is deprecated since sonata-project/admin-bundle 3.x'
+                .' and will not be used in version 4.0. Use configureDashboardActions() instead.'
+            );
```

Comment:
```suggestion
            @trigger_error(sprintf(
                'The "%s::configureDashboardButtons()" method is deprecated since sonata-project/admin-bundle 3.x'
                .' and will not be used in version 4.0. Use "configureDashboardActions()" instead.'
                __CLASS__
            ), \E_USER_DEPRECATED);
```

Reply:
Done